src/services/coreData/base.ts

```diff
@@ -187,6 +189,24 @@ class Base {
     return manifests;
   }
 
+  /**
+   * Returns the related media contents for the record with the passed ID.
+   *
+   * @param id
+   */
+    async getRelatedMediaContents(id: string) {
```

Comment:
Do we also need to update the loaders for this to work in "static" mode?

Reply:
Looks like #186 implements the Core Data loaders, can we pull `develop` into this branch once that PR is merged and confirm that this will work in "static" mode?

Reply:
Done!